src/components/IntegrationTests/IntegrationTestForm/IntegrationTestView.tsx

```diff
@@ -55,10 +56,20 @@ const IntegrationTestView: React.FunctionComponent<
   interface FormContext {
     name: string;
     description: string;
+    selected?: boolean;
   }
 
-  const getFormContextValues = (contexts: Context[] | null | undefined): FormContext[] => {
-    if (!contexts?.length) return [];
+  const getFormContextValues = (
+    integrateTest: IntegrationTestScenarioKind | null | undefined,
+  ): FormContext[] => {
```

Comment:
Adding unit tests for this utility should help with the codecoverage

Reply:
Good catch! Test added.